omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_2/LivingPatientSearchHandler2_2Test.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+
+package org.openmrs.module.webservices.rest.web.v1_0.search.openmrs2_2;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.api.context.Context;import org.openmrs.module.webservices.rest.web.RestTestConstants1_8;import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.RestControllerTestUtils;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+
+
+import java.util.List;
+
+
+public class LivingPatientSearchHandler2_2Test extends  RestControllerTestUtils  {
+
+	protected String getURI() {
+		return "patient";
+	}
+
+	
+	@Test
+	public void getSearchConfig_shouldReturnAllPatients() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI());
+		req.addParameter("includeDead", "true");
+		
```

Comment:
All your tests return the same number of patients. I thought the results would be different for all, dead, and living patients.

Reply:
ya , in the dataset there was currently no dead patient ..ill just add it

Reply:
I have updated my tests to include the dead patient

Reply:
I have included a dead patient , for the tests